### PR TITLE
add options to mount read-only and set the fuse-mount-option allow_other

### LINKED ISSUE #13: mount option is "rw"
As far as I understand the filesystem mounted with ipfs-api-mount is completely read-only.

But the mount point is set to 'rw'.

This is leading to unnecessary requests hitting your python code (I guess).

### PR DESCRIPTION
**Pull request by RubenKelevra**
fixes #13

### CODE REVIEW

File: bin/ipfs-api-mount [hunk 1]
```diff
@@ -18,6 +18,8 @@ if __name__ == '__main__':
     parser.add_argument('--object-links-cache-size', type=int, default=256, help='Max number of object link sections kept in cache.')
     parser.add_argument('-b', '--background', action='store_true', help='Run in background.')
     parser.add_argument('-m', '--multithreaded', action='store_true', help='Use multiple threads to handle FS requests.')
+    parser.add_argument('--allow-other', action='store_true', help='Set fuse mount option \'allow_other\'')
+    parser.add_argument('--read-only', action='store_true', help='Set \'ro\' mount option instead of \'rw\'')
```
Comment (SupraSummus) [inline bin/ipfs-api-mount]: I feel like `ro` should be always set, and theres no need to expose it as a parameter.

**Comment by SupraSummus** [comment]
Thanks for contribution.

I've rebased and merged it locally. Also, I've removed the read-only parameter. Now the mount is always read-only.

**Comment by RubenKelevra** [comment]
> Thanks for contribution.
> 
> I've rebased and merged it locally. Also, I've removed the read-only parameter. Now the mount is always read-only.

I wasn't sure about any plans to do write requests in the future, that's the reason for exposing it. :)

**Comment by SupraSummus** [comment]
I thought of implementing write-enabled mount point, indeed. But I'm not going to write this any time soon. PR's are welcome ;D

**Pull request closed without merging**